app/code/Magento/CatalogImportExport/Model/Import/Uploader.php

```diff
@@ -161,6 +161,15 @@ public function move($fileName, $renameFileOff = false)
                 $read = $this->_readFactory->create($url, DriverPool::HTTPS);
             }
 
+            //only use filename (for URI with query parameters)
+            $parsedUrl = parse_url($url);
```

Comment:
you could use the optional component parameter of parse_url here
```
$parsedUrl = parse_url($url, PHP_URL_PATH);
if ($parsedUrl) {
```

Reply:
True. I edited my code.